runtime/fail.js

```diff
@@ -26,6 +26,9 @@ function caml_return_exn_constant (tag) { return tag; }
 //Provides: caml_raise_with_arg (const, const)
 function caml_raise_with_arg (tag, arg) { throw [0, tag, arg]; }
 
+//Provides: caml_raise_with_args (const, const, const)
+function caml_raise_with_args (tag, nargs, args) { throw [0, tag].concat(args); }
```

Comment:
Can we validate, maybe truncate `args` based on `nargs` ? Alternatively, we could completely remove `nargs`, this function is only a helper for writing stubs

Reply:
Oh, so the function that exists in ocaml core isn't used anywhere except for external stubs? Would JSOO want to provide the same interface that it provides? I'm happy to add a validation or truncate.

Reply:
I don't think we need to keep the same interface in the case. I would vote for removing the unused argument